cancer_ai/validator/rewarder.py

```diff
@@ -61,7 +61,7 @@ def add_score(self, competition_id: str, hotkey: Hotkey, score: float, date: dat
         self.score_map[competition_id][hotkey].sort(key=lambda x: x.date)
         if len(self.score_map[competition_id][hotkey]) > HISTORY_LENGTH:
             # remove the oldest one
-            self.score_map[competition_id][hotkey] = self.score_map[competition_id][hotkey][1:]
+            self.score_map[competition_id][hotkey] = self.score_map[competition_id][hotkey][-HISTORY_LENGTH:]
```

Comment:
the same logic, but  if there will be LENGTH+2, then it won't trim to the size. and we need it to trim properly for changing history size